install/kubernetes/quick-install.yaml

```diff
@@ -508,10 +549,6 @@ spec:
           mountPropagation: HostToContainer
         - mountPath: /var/run/cilium
           name: cilium-run
-        resources:
-          requests:
-            cpu: 100m
-            memory: 100Mi
```

Comment:
I think it likely still makes sense to have these requests for the `clean-cilium-state` container.

Reply:
done thanks